### DISCUSSION TITLE
Compatibility with my system

### DISCUSSION THREAD

**Discussion by Oxygen54** [JanM321/esphome-lg-controller#73] [General]
Hi,

I have a LG unit on a ducted heating and this system use a modbus converter (RS485) (INTESIS LG-RC-MBS-1) to control the AC unit, I have a LG PREMTB100 linked too as a slave. 

I want to control the system with this project, did I need to use the modbus converter or use the schematic on this project?

Thanks

**Comment by JanM321** [comment]
You could use this project to replace the PREMTB100.

It might work better to replace both the modbus converter and the PREMTB100 and just connect this ESPHome controller to the AC's Red/Yellow/Black CN-REMO port.
